### PR TITLE
simplify code and update docs

### PR DESCRIPTION
**Pull request by septs**
plz update `1.0.5` to this commit hash. thanks.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -22,11 +22,10 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Run gcov2lcov
+    - shell: bash
       run: ${{ github.action_path }}/entrypoint.sh
-      shell: bash
+      working-directory: ${{ inputs.workspace }}
```
Comment (jandelgado) [inline action.yml]: * please document `workspace` input in the `README`
* what happens if the `workspace` input is NOT set (it is not required)?

**Review by jandelgado — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (septs) [reply action.yml]: the repo use case (test data in subdirectory)

maybe need monorepo this feature (change working directory)

Comment (septs) [reply action.yml]: in fact, we are all is optional inputs.

see https://github.com/jandelgado/gcov2lcov-action/blob/4b58d89f12597f97e8c509dfbcbb6a71a3dde8fa/.github/workflows/test.yml#L20-L23

Comment (septs) [reply action.yml]: > * what happens if the `workspace` input is NOT set (it is not required)?

default case working directory is `$GITHUB_WORKSPACE`

**Comment by septs** [comment]
@jandelgado

**Review by jandelgado — APPROVED** [review]

**Pull request merged**